docs/relational-databases/backup-restore/restore-a-sql-server-database-to-a-point-in-time-full-recovery-model.md

```diff
@@ -150,7 +150,7 @@ ms.author: chadam
   
 3.  Restore the last differential database backup, if any,  without recovering the database (RESTORE DATABASE *database_name* FROM *backup_device* WITH NORECOVERY).  
   
-4.  Apply each transaction log backup in the same sequence in which they were created, specifying the time at which you intend to stop restoring log (RESTORE DATABASE *database_name* FROM <backup_device> WITH STOPAT**=**_time_**,** RECOVERY).  
+4.  Apply each transaction log backup in the same sequence in which they were created, specifying the time at which you intend to stop restoring log (RESTORE DATABASE *database_name* FROM <backup_device> WITH STOPAT **=** _time_ **,** RECOVERY).  

```

Comment:
@cawrites

I wonder if we should try a more standard way to identify text that needs ot be replaced. ```suggestion
4.  Apply each transaction log backup in the same sequence in which they were created, specifying the time at which you intend to stop restoring log (`RESTORE DATABASE <database_name> FROM <backup_device> WITH STOPAT = <time>, RECOVERY`).  
```